### ISSUE TITLE
set up monitoring for DB consistency

### ISSUE THREAD

**Issue by mapreri** [paulproteus/public-udd-mirror#17]
I was thinking of having a script run on udd.d.o, that regularly polls udd-mirror for:

 1. freshness of the dump (guess checking `https://udd-mirror.debian.net/logs/stamp.txt` should do?)
 2. do a couple of `select count(*)` in, like, `upload_history` and somewhere else to check the db is sane

else it would start spamming us.

**Comment by intrigeri** [comment]
Yes, it would be nice if such monitoring was in place! (Otherwise, if the DB update is broken, users _eventually_ notice side-effects and report issues such as #24.)
